src/components/registration/RegistrationForm.unit.test.tsx

```diff
@@ -71,18 +85,20 @@ const TEST_GUEST_DYNAMIC_FIELD = {
 };
 
 describe("when text fields are submitted", () => {
-  it("structures the data in the required format", () => {
+  it("structures the data in the required format", async () => {
```

Comment:
i think we removed the `await` calls we were playing with earlier, so we can remove this